tuxemon/core/monster.py

```diff
@@ -266,7 +262,7 @@ def load_from_db(self, slug):
         """
 
         # Look up the monster by name and set the attributes in this instance
-        results = monsters.lookup(slug)
+        results = db.lookup(slug)
```

Comment:
results = db.lookup(slug, table="monster")

Reply:
Ouch... thanks for catching that.